<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblAnswer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>650</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>650</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>650</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>632</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="btnZero">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(68, 68, 68);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
    background-color: #5C5C5C;
}
</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnPlusMinus">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(184, 184, 184);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: black;
}
QPushButton:hover{
	background-color: rgb(217, 217, 217);
}</string>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="btnTan">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>28</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 159, 10);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
	background-color: rgb(251, 199, 142);
}
</string>
       </property>
       <property name="text">
        <string>tg</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnNine">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(68, 68, 68);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
    background-color: #5C5C5C;
}</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btnThree">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(68, 68, 68);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
    background-color: #5C5C5C;
}
</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="btnMClear">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>28</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 159, 10);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
	background-color: rgb(251, 199, 142);
}
</string>
       </property>
       <property name="text">
        <string>MC</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnDiv">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 159, 10);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
	background-color: rgb(251, 199, 142);
}
</string>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btnOne">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(68, 68, 68);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
    background-color: #5C5C5C;
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btnPlus">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 159, 10);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
	background-color: rgb(251, 199, 142);
}
</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="btnSin">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>28</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 159, 10);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
	background-color: rgb(251, 199, 142);
}
</string>
       </property>
       <property name="text">
        <string>sin</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnDel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>28</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(184, 184, 184);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: black;
}
QPushButton:hover{
	background-color: rgb(217, 217, 217);
}</string>
       </property>
       <property name="text">
        <string>DEL</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btnSeven">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(68, 68, 68);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
    background-color: #5C5C5C;
}</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnMultiple">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 159, 10);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
	background-color: rgb(251, 199, 142);
}
</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btnMinus">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 159, 10);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
	background-color: rgb(251, 199, 142);
}
</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="btnDot">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(68, 68, 68);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
    background-color: #5C5C5C;
}</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="btnCos">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>28</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 159, 10);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
	background-color: rgb(251, 199, 142);
}
</string>
       </property>
       <property name="text">
        <string>cos</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btnTwo">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(68, 68, 68);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
    background-color: #5C5C5C;
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btnFour">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(68, 68, 68);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
    background-color: #5C5C5C;
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="btnCtan">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>28</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 159, 10);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
	background-color: rgb(251, 199, 142);
}
</string>
       </property>
       <property name="text">
        <string>ctg</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnFive">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(68, 68, 68);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
    background-color: #5C5C5C;
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btnSix">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(68, 68, 68);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
    background-color: #5C5C5C;
}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnEight">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(68, 68, 68);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
    background-color: #5C5C5C;
}</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btnAC">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(184, 184, 184);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: black;
}
QPushButton:hover{
	background-color: rgb(217, 217, 217);
}</string>
       </property>
       <property name="text">
        <string>AC</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="btnEqual">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 159, 10);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
	background-color: rgb(251, 199, 142);
}
</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="btnMRead">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>28</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 159, 10);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
	background-color: rgb(251, 199, 142);
}
</string>
       </property>
       <property name="text">
        <string>MR</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="btnMMinus">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>28</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 159, 10);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
	background-color: rgb(251, 199, 142);
}
</string>
       </property>
       <property name="text">
        <string>M-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="btnSqrt">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>28</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 159, 10);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
	background-color: rgb(251, 199, 142);
}
</string>
       </property>
       <property name="text">
        <string>sqrt</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="btnMPlus">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>28</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 159, 10);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
	background-color: rgb(251, 199, 142);
}
</string>
       </property>
       <property name="text">
        <string>M+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="btn1x">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>28</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 159, 10);
	font: 28pt &quot;Segoe UI&quot;;
	border-radius: 50%;
	color: white;
}
QPushButton:hover{
	background-color: rgb(251, 199, 142);
}
</string>
       </property>
       <property name="text">
        <string>1/x</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
